marxs/source/labSource.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+from astropy.table import Table, Column
+
+from .source import Source
+
+
+class FarLabConstantPointSource(Source):
+	'''Simple in-lab source for testing purposes
+
+	- point source
+	- TODO: figure out how to provide energy distribution
+	- photons uniformly distributed through rectangular space (reasonable approximation if source is far)
+	- photon direction determined by location of source, and selected photon starting position
+	'''
+	def __init__(self, position, polarization, rate, energyMean, energyStdDev):
+		self.pos = position
+		self.polar = polarization
+		self.rate = rate   # photons/sec
+		self.meanEnergy = energyMean
+		self.stdDevEnergy = energyStdDev
+
+	def generate_photons(self, t, center, v_y, v_z):
```

Comment:
Or maybe you can move these parameters into `init`, just like the `position`, etc.?